src/devices/CharacterLcd/Hd44780.cs

```diff
@@ -63,7 +63,7 @@ public class Hd44780 : IDisposable
         private byte _lastByte;
         private bool _useLastByte;
 
-        private (int pin, PinValue value)[] _pinBuffer = new (int, PinValue)[8];
+        private PinValuePair[] _pinBuffer = new PinValuePair[8];
```

Comment:
I wonder if this should just be a byte and offset in memory

Reply:
This is just a buffer for sending the sequence of pins. I could do this on the stack and still might as it would be local to the operation. The set is 64 bytes.